src/ciphersuite/mod.rs

```diff
@@ -562,6 +570,7 @@ impl AeadKey {
         AeadKey {
             aead_mode,
             value: aead_key_gen(aead_mode),
+            mac_len: 16,
```

Comment:
We could pass `&Ciphersuite` as an argument here as well in order to avoid the hard-coding

Reply:
Just added it ;)